Web/ExamGenerator.Web/Areas/Administration/Views/QuestionType/Create.cshtml

```diff
@@ -0,0 +1,24 @@
+﻿@model QuestionTypeInputModel
+
+@{
+    ViewData["Title"] = "QuestionType Create";
+}
+
+<h1>@ViewData["Title"]</h1>
+<div class="row">
+    <div class="col-md-4">
+        <section>
+            <form id="account" method="post">
+                <div asp-validation-summary="All" class="text-danger"></div>
+                <div class="form-group">
+                    <label asp-for="Name"></label>
+                    <input asp-for="Name" class="form-control" />
+                    <span asp-validation-for="Name" class="text-danger"></span>
+                </div>
```

Comment:
You shoul have dropdown to choose subject. You can use Component like this example: https://github.com/NaskoVasilev/JudgeSystem/blob/master/Web/JudgeSystem.Web/Views/Shared/Components/CoursesSelectList/Default.cshtml
https://github.com/NaskoVasilev/JudgeSystem/blob/master/Web/JudgeSystem.Web/Components/CoursesSelectListViewComponent.cs
In this way you can use it in the view: <vc:courses-select-list></vc:courses-select-list>